package/MDAnalysis/core/groups.py

```diff
@@ -103,6 +103,7 @@
 from ..lib import distances
 from ..lib import transformations
 from ..lib import mdamath
+from ..lib.accessors import Accessor, ConverterWrapper
```

Comment:
Let's make a new top level `MDAnalysis.converters` 
```suggestion
from ..converters.accessors import Accessor, ConverterWrapper
```

Reply:
Should I move the actual converter classes that are currently in `MDAnalysis.coordinates` in there as well then ? It would make more sense to me.

Reply:
Would make more sense to me too.

Reply:
Yep